### ISSUE TITLE
Find a new primary anime provider

### ISSUE THREAD

**Issue by vn-ki** [vn-ki/anime-downloader#25]
9anime has been unreliable for the past week. A new fix has emerged but it would probably need running javascript on the machine unless someone decides to run a server with that code. 

Ref: https://github.com/DxCx/plugin.video.9anime/issues/267

The best way for us would be to move onto a different primary provider. I am asking you guys for suggestions for the new provider because I generally watch old anime and I don't know which websites get the new ones first.

I am thinking of making gogoanime primary after making it more robust (Right now it has lots of bugs).

Please note the suggestion **should not** be cloudflare secured.

**Comment by Karma32** [comment]
https://otakuplan.com/blogs/news/best-32-websites-to-watch-and-stream-anime-online-in-2018
This might help

**Comment by MobileSuitSadBoi** [comment]
I've tried a lot of different add-ons and none of them come close to 9anime. It has pretty much anything I can think of old and new. Plus it's updated often. I've been using squad control as a backup but they don't have everything I look for. Ex. They don't have the subbed version of neon Genesis only the dubbed. Cringe worthy.

**Comment by RedNinjaX** [comment]
I've already figured out how to grab content from Vrv (Crunchyroll and Funimation) as a start. A few examples can be seen here: https://www.wonderfulsubs.com

**Comment by ghettowboy** [comment]
@vn-ki I have code for gogoanime and Masterani if you want to look at it? Lol 

@RedNinjaX I don't think you can pull from CR and/or Funi fully, they have free content but they require membership to view updated episodes+. Unless VRV itself is a exploit... ?

**Comment by vn-ki** [comment]
@ghettowboy Yes. I'd be interested in your code. I have partially working gogo and masterani. I never completed them because 9a was so good. It would awesome if you could share your code.

@RedNinjaX Wow. You really are something, man. It's awesome that you're doing all this. I'm super interested in your site. Looking forward to when it's completed. I hope it'll be a free service. (I don't mind non intrusive ads)

**Comment by RedNinjaX** [comment]
@ghettowboy thanks. And yes I have no intentions on ads or charging for anything ;)

I do hope to open up my API for select use to other people. It contains everything from the metadata for each episode to the urls for each server

**Comment by MobileSuitSadBoi** [comment]
Can you provide instructions on how to download masterani add-on please. I've tried 6 times now and  anywhere I go I can't seem to find it

**Comment by RedNinjaX** [comment]
@vn-ki thanks. And yes I have no intentions on ads or charging for anything ;)

I do hope to open up my API for select use to other people. It contains everything from the metadata for each episode to the urls for each server

**Comment by ghettowboy** [comment]
@vn-ki @RedNinjaX ..... Here's the code, hope it helps!
[requested.zip](https://github.com/vn-ki/anime-downloader/files/2130334/requested.zip)

**Comment by RedNinjaX** [comment]
@ghettowboy thanks!

**Comment by ghettowboy** [comment]
@MobileSuitSadBoi yeah, addons for Anime are really hard to find. unfortunately the Masterani addon stopped working about 5 months ago (needs update), the gogoanime is old but still works as is (it's what I'm currently using now). Anyways NOW when I come across addons I keep them stored.

**Comment by poiuedws** [comment]
you could use twist.moe
